tests/director/tests/runtime_contexts_api_test.go

```diff
@@ -164,8 +164,8 @@ func TestRuntimeContextSubscriptionFlows(stdT *testing.T) {
 	t.Run("Runtime Contexts subscription flows", func(t *testing.T) {
 		ctx := context.Background()
 		subscriptionProviderSubaccountID := conf.TestProviderSubaccountID // the parent is testDefaultTenant
-		subscriptionConsumerAccountID := tenant.TestTenants.GetIDByName(t, tenant.ApplicationsForRuntimeTenantName)
-		subscriptionConsumerSubaccountID := conf.TestConsumerSubaccountID // the parent is ApplicationsForRuntimeTenantName
```

Comment:
I would suggest leaving the comment with the parent only to emphasize it is for the local setup.